{{cookiecutter.project_name}}/.github/next_steps/05_linting.md

```diff
@@ -2,4 +2,13 @@
 title: 'Next step: Linting'
 ---
 
-Linting instructions
+Your repository has a [workflow]({{ cookiecutter.repository }}/blob/main/.github/workflows/lint.yml) which [lints](https://en.wikipedia.org/wiki/Lint_(software)) your code in every push and when creating a pull request.
+
+To make sure you have a proper linter configuration you need to follow the steps below.
+
+1. Update `description` and `keywords` fields in [setup.cfg]({{ cookiecutter.repository }}/blob/main/setup.cfg).
```

Comment:
How do I need to update it? Run prospector first

Reply:
updated the text